lib/fastlane/actions/sigh.rb

```diff
@@ -9,22 +10,25 @@ def self.run(params)
         require 'sigh'
         require 'credentials_manager/appfile_config'
 
-        type = Sigh::DeveloperCenter::APPSTORE
-        type = Sigh::DeveloperCenter::ADHOC if params.include? :adhoc
-        type = Sigh::DeveloperCenter::DEVELOPMENT if params.include? :development
+        type = FastlaneCore::DeveloperCenter::APPSTORE
+        type = FastlaneCore::DeveloperCenter::ADHOC if params.include? :adhoc
+        type = FastlaneCore::DeveloperCenter::DEVELOPMENT if params.include? :development
+        force = params.include? :force
 
         return type if Helper.test?
 
         app = CredentialsManager::AppfileConfig.try_fetch_value(:app_identifier)
         raise 'No app_identifier definied in `./fastlane/Appfile`'.red unless app
 
-        path = Sigh::DeveloperCenter.new.run(app, type)
+        CredentialsManager::PasswordManager.shared_manager(ENV['SIGH_USERNAME']) if ENV['SIGH_USERNAME']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.